client-linuxapp/src/main.rs

```diff
@@ -446,37 +484,62 @@ async fn main() -> Result<()> {
     }
 
     // Get rv entries
-    let rv_info = get_rv_info(dc.as_ref());
-
-    let client_list = get_client_list(rv_info.unwrap())
-        .await
-        .context("Error getting usable rendezvous client")?;
+    let rv_info = get_rv_info(dc.as_ref())?;
+    let mut rv_info_it = rv_info.iter().cycle();
 
-    // Get owner info
-    let to1d = get_to1d(dc.as_ref(), client_list).await?;
+    let mut rv_entries_processed: usize = 0;
+    loop {
+        let rv_entry = rv_info_it
+            .next()
+            .ok_or_else(|| anyhow!("No RV entries returned from the iterator"))?;
 
-    let to1d_payload: UnverifiedValue<TO1DataPayload> = to1d
-        .get_payload_unverified()
-        .context("Error getting the TO2 payload")?;
-    let to2_addresses = to1d_payload.get_unverified_value().to2_addresses();
-    let to2_addresses = get_to2_urls(to2_addresses);
-    log::info!("Got TO2 addresses: {:?}", to2_addresses);
+        let client_list = get_client_list(rv_entry)
+            .await
+            .context("Error getting usable rendezvous client")?;
```

Comment:
This will return an `Error` if an RV Entry only specifies wifi or delay, and no URL.

Reply:
I understand that with [this](https://github.com/fedora-iot/fido-device-onboard-rs/pull/41/files#r693861156), it'd be addressed, am I right?

Reply:
Yep, thanks.